grafana/resource_data_source.go

```diff
@@ -193,6 +194,11 @@ func ResourceDataSource() *schema.Resource {
 							Optional:  true,
 							Sensitive: true,
 						},
+						"private_key": {
```

Comment:
sorry to sound like a broken record, could we also keep these in alpha order?

Reply:
sure thing!